CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.7.0 (2019-01-27)
+
+- Added the fuzzy date pipe. [#100](https://github.com/blackbaud/skyux-datetime/pull/100)
```

Comment:
Should this link to https://github.com/blackbaud/skyux-datetime/pull/102? When I follow the URL we have in place now, it opens an issue instead of a PR. ... Otherwise, this looks good to me.

Reply:
LGTM other than that and the fact that the build is failing for some reason

Reply:
Oops, good catch. fixed.